src/main.cpp

```diff
@@ -150,15 +150,15 @@ int main(int argc, char **argv)
             i++;
             break;
         case 's':
-            snaplen = atoi(argv[i + 1]);
+            snaplen = strtol(argv[i + 1], NULL, 10);
```

Comment:
>strtol [](http://example.com/codeflow?start=22&length=6)

This new function has better error handling, you may consider add more code to check for error cases, and report back to user.